samcli/lib/docs/documentation_links.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": "https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-config.html",
```

Comment:
There are a lot of repetitive URLs here but I left it this way since we have no way of guaranteeing that new URLs will have the same pattern. It also just keeps the configuration easy instead of trying to parse patterns for different types of URLs in our documentation.